src/models/mrt/mod.rs

```diff
@@ -99,6 +100,7 @@ impl CommonHeader {
     }
 }
 
+// TODO: Why isn't there an option for an unknown message?
```

Comment:
Never encountered unknown message types before. The completeness, we may add a `Unknown` message, but it will probably not provide much value.

Reply:
Yea, if we don't know the MRT type then there isn't exactly much that can even be parsed. I'm not sure what my reasoning was behind this particular comment.